### PR TITLE
Add JSON server

### PR DESCRIPTION
**Pull request by tlorent**

### CODE REVIEW

File: services/fetch-timesheets/fetch-timesheets.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+const fetchTimesheets = () => fetch('http://localhost:3001/timesheets')
```
Comment (larstroost) [inline services/fetch-timesheets/fetch-timesheets.js]: url: http://localhost:3001/api/timesheets

File: services/fetch-timesheets/fetch-timesheets.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+const api = 'http://localhost:3001/api/timesheets';
```
Comment (RobbinHabermehl) [inline services/fetch-timesheets/fetch-timesheets.js]: This filename implies you'll create a new service for other operations; it's better to create one service (`TimesheetsAPi` for instance) for all.

File: services/TimesheetsApi/TimesheetsApi.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+const api = 'http://localhost:3001/api/timesheets';
```
Comment (RobbinHabermehl) [inline services/TimesheetsApi/TimesheetsApi.js]: Only components should have capitals in their filename, this should become `timesheets-api.js`.

Comment (tlorent) [reply services/TimesheetsApi/TimesheetsApi.js]: Got it, fixed!

**Pull request merged**